pkg/transforms/common.go

```diff
@@ -548,9 +548,14 @@ func edgesToApplication(nodeInfo NodeInfo, ns NodeStore, UID string, onlyApplica
 }
 
 type relatedObject struct {
-	kind      string
-	namespace string
-	name      string
+	apiVersion string
+	kind       string
+	namespace  string
+	name       string
+}
+
+func (r relatedObject) String() string {
+	return r.apiVersion + " " + r.kind + " " + r.namespace + " " + r.name
```

Comment:
Should we use a format similar to the kube api for the resource? For example:
```
/namespace/apigroup/apiversion/kind/name
```
Also namespace will be empty for cluster-scoped resources.

Reply:
I'd rather the string format just be for sorting and testing, and have a more structured way to pass the data around, but I need to look into the details on why it wasn't getting to the console the way I was expecting yet.

Reply:
It's possible that the search-api is trying to parse the string like the labels list.  I'd have to debug to confirm.

The uri string can be sorted for testing. But lets check what we do in `argoapplications.go` for consistency.

Reply:
I'll work on matching `argoapplications.go` :+1: